public/app/features/dashboard/components/TransformationsEditor/SqlExpressionCard.tsx

```diff
@@ -1,21 +1,32 @@
-import { css } from '@emotion/css';
+import { css, cx } from '@emotion/css';
 
 import { GrafanaTheme2 } from '@grafana/data';
-import { Card, useStyles2 } from '@grafana/ui';
+import { Card, IconButton, useStyles2 } from '@grafana/ui';
 
 export interface SqlExpressionCardProps {
   name: string;
   description: string;
   imageUrl?: string;
   onClick: () => void;
   testId?: string;
+  isDisabled?: boolean;
+  disabledTooltip?: string;
 }
 
-export function SqlExpressionCard({ name, description, imageUrl, onClick, testId }: SqlExpressionCardProps) {
+export function SqlExpressionCard({
+  name,
+  description,
+  imageUrl,
+  onClick,
+  testId,
+  isDisabled,
+  disabledTooltip,
+}: SqlExpressionCardProps) {
   const styles = useStyles2(getSqlExpressionCardStyles);
+  const cardClasses = isDisabled ? cx(styles.card, styles.cardDisabled) : styles.card;
 
   return (
-    <Card className={styles.card} data-testid={testId} onClick={onClick} noMargin>
+    <Card className={cardClasses} data-testid={testId} onClick={onClick} noMargin>
```

Comment:
Nit (non-blocking): You could inline the class names if you want to make it clean 😄 You'd just need to delete `cardClasses` above if so.

```suggestion
    <Card
      className={cx(styles.card, { [styles.cardDisabled]: isDisabled })}
      data-testid={testId}
      onClick={onClick}
      noMargin
    >

```